doc/CHANGES.md

```diff
@@ -2,6 +2,11 @@
 
 ## Highlighted Features
 
+* New statistics page that includes user stats to show top requesters and
+  annotators, and hidden  requests. Includes a new event type of "hide" to
+  make tracking and reporting on hidden requests much simpler. Need to run
+  `rake temp:update_hide_event_type` to set up the data for this feature
+  (Henare Degan)
```

Comment:
Can we please change this to `(Henare Degan, Luke Bacon)`. Although the commits are attributed to me we paired on this entire feature.